index.html

```diff
@@ -217,6 +217,41 @@ <h2>Languages</h2>
           </div>
         </div>
       </section>
+      <section class="contact-form">
+        <h3>Contact me</h3>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      <form action="https://formspree.io/f/mayzpyer" method="post">
+        <input
+          class="form"
+          type="text"
+          name="name"
+          placeholder="Name"
+          maxlength="30"
+          required
+        />
+        <input
+          class="form"
+          type="email"
+          name="email"
+          placeholder="Email"
+          maxlength="30"
+          required
+        />
+        <textarea
+          name="message"
+          class="textarea"
+          placeholder="Enter your message here..."
+          cols="30"
+          rows="10"
+          maxlength="500"
+          required
+        ></textarea>
+        <button type="submit">Get in touch</button>
```

Comment:
- [ ] Good job adding the hover interaction 👍👍 However, as suggested by the reviewer [here](https://github.com/SolomonTheCoder/Microverse-Portfolio/pull/3#discussion_r1150909579) please make sure that your button interactions also follow the design guidelines such as `enabled`, `hover`, `pressed` and `disabled`⚒ As an example, see the images below of `hover` and `active` interactions for more clarity 👇

### _Hover interaction 🔽_
Your design before | Your design now | Figma design
---| ---| ---| 
![image](https://user-images.githubusercontent.com/106119105/228342571-896ea284-9e37-4485-8dcb-bc79b8f30e20.png) | ![image](https://user-images.githubusercontent.com/106119105/228342632-e159f861-09f4-4c68-8412-a252a2ee24a3.png) | ![image](https://user-images.githubusercontent.com/106119105/228343038-17d2c9a9-86d2-409f-8091-3e3cad1385c3.png)


### _Active interaction 🔽_
Your design before | Your design now | Figma design
---| ---| ---| 
![image](https://user-images.githubusercontent.com/106119105/228343215-fd02bfb2-0b65-4cb5-a113-62847a52495f.png) | ![image](https://user-images.githubusercontent.com/106119105/228343408-99afd443-785b-440a-8b5a-20a3dba95d66.png) | ![image](https://user-images.githubusercontent.com/106119105/228343461-bc3d2554-4725-4a83-a47c-e50c240a23bb.png)